package.json

```diff
@@ -76,8 +77,5 @@
   },
   "lint-staged": {
     "**/*": "prettier --write --ignore-unknown"
-  },
-  "engines": {
-    "node": "16"
   }
```

Comment:
I'm not sure this is necessary? I was able to get it installed and running with the current version of node (v21.5.0)

Reply:
It's not. Hacky fix to get it to build on railway.app. Will open an issue to find a longer term solution